### PR TITLE
Fix #2515: Updated literals to named arguments in TopicLessonsFragmentTest.kt.

### LINKED ISSUE #2515: Update literals to named arguments in TopicLessonsFragmentTest.kt
**Is your feature request related to a problem? Please describe.**
Update the literals passed as argument to named arguments. This improves code readability and provides better context to the readers. One such instance is shown as an example below.

**Describe the solution you'd like**
Update the argument in line 246 below:
https://github.com/oppia/oppia-android/blob/bb7351b4c0700b3dadd1a51edd99e8e51c696eb4/app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt#L244-L248

to the following code:
```        
        atPositionOnView(
          R.id.story_summary_recycler_view,
          position = 1,
          R.id.chapter_recycler_view
        )
```

**Additional context**
1. Identify other instances of such changes in `TopicLessonsFragmentTest`, update the code with named arguments. 
2. Identify other files/classes where such changes, update with named arguments for better readability.

### PR DESCRIPTION
**Pull request by harshpatel63**
## Explanation
 Fixes #2515 . All the literals in the file has been updated to named arguments.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 1]
```diff
@@ -123,7 +123,8 @@ class TopicLessonsFragmentTest {
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
```
Comment (srikanthkb) [inline app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: Let's revert the change in this line, as it is not required to explicitly mention class name for "this"

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt [hunk 2]
```diff
@@ -124,7 +124,7 @@ class TopicLessonsFragmentTest {
 
   private fun setUpTestApplicationComponent() {
     ApplicationProvider.getApplicationContext<TestApplication>()
-      .inject(topicLessonsFragmentTest = this)
+      .inject(this)
```
Comment (srikanthkb) [inline app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: This can be accommodated on a single line (as in the current codebase).

**Review by srikanthkb — APPROVED** [review]
LGTM. Thanks

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 1 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: @Sci3fic please reply to all comments.

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 2 above)
Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: @Sci3fic please reply to all comments.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Comment by rt4914** [comment]
Will merge this PR once @Sci3fic Replies to the comments and @Srikanth-Kb resolves the comment afterwards.

**Comment by harshpatel63** [comment]
> Will merge this PR once @Sci3fic Replies to the comments and @Srikanth-Kb resolves the comment afterwards.

Glad to see my first contribution with this great organisation.☺️

**Comment by srikanthkb** [comment]
> > Will merge this PR once @Sci3fic Replies to the comments and @Srikanth-Kb resolves the comment afterwards.
> 
> Glad to see my first contribution with this great organisation.☺️

You can reply as simply as "Done" for all the review comments you receive. So, that we will be able to resolve the conversations after review.

File: app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt (hunk 1 above)
Comment (harshpatel63) [reply app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt]: Done.

**Comment by srikanthkb** [comment]
Thanks @Sci3fic. Let us know, if there are other issues which interests you. 

@rt4914 We are good to go. :)
